terraform/modules/google/compute/nodejs/nodejs.sh.tpl

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
```

Comment:
At some point we should consolidate these scripts into a common directory so they're not duplicated, but let's leave it for now.